contributors/devel/running-locally.md

```diff
@@ -83,7 +83,7 @@ cd kubernetes
 Since root access is sometimes needed to start/stop Kubernetes daemons, `./hack/local-up-cluster.sh` may need to be run as root. If it reports failures, try this instead:
 
 ```sh
-sudo ./hack/local-up-cluster.sh
+sudo -E PATH=$PATH ./hack/local-up-cluster.sh
```

Comment:
this can also _break_ users, imagine if they only have docker available to root (which is why we need to sudo) ... this is dependent on users and their local path manipulation ...

what was missing from PATH in your case?

Reply:
etcd was missing -- if one follows the instructions earlier on the page to install etcd, it gets installed in such a way that you have to manually add it to your $PATH

Reply:
Ah, I think we should probably update the script to find etcd in the expected install location in addition to $PATH.

Reply:
> My PATH includes etcd, because that is also needed for other commands (go test ./test/integration/<something>).

Well, the integration scripts actually handle this for you now :-)

We could have the go targets do it too, it will just be messier.

but either way, if you run with sudo at the top level your path will not be preserved

we should probably stop specifying sudo here entirely

Reply:
Did a little more testing and I confirmed that it works without sudo for me.

I figured out why I thought I needed the sudo workaround though: after I've run the script once, before I can run it again I have to manually delete /var/run/kubernetes/* and the relevant /tmp/*.log files or it fails to come up -- it looks like it's because a smattering of these files end up owned by root which breaks things.

Reply:
That observation is correct. I forgot to mention that I am doing the same manual cleanup.

It would be good to fix that in the script itself so that it can be invoked repeatedly.